diff --git a/services/sync/sync.properties b/services/sync/sync.properties
--- a/services/sync/sync.properties
+++ b/services/sync/sync.properties
@@ -3,10 +3,10 @@
 
 # %S is the date and time at which the last sync successfully completed
 lastSync.label = Ostatnia aktualizacja: %S
-lastSyncInProgress.label = Ostatnia aktualizacja. Pobieranie danych…
+lastSyncInProgress.label = Ostatnia aktualizacja – pobieranie danych…
 
 # %S is the username logged in
 connected.label = Połączono: %S
 disconnected.label = Rozłączono
 connecting.label = Łączenie…
 
@@ -7,10 +7,10 @@
 
 # %S is the username logged in
 connected.label = Połączono: %S
 disconnected.label = Rozłączono
 connecting.label = Łączenie…
 
-mobile.label = Zakładki mobilne
+mobile.label = Zakładki urządzenia przenośnego
 
 remote.pending.label = Synchronizowanie zdalnych kart…
 remote.missing.label = Zsynchronizuj swoje inne urządzenia ponownie, by uzyskać dostęp do ich kart
@@ -18,8 +18,8 @@
 remote.notification.label = Po synchronizacji będą dostępne ostatnio używane karty komputera stacjonarnego
 
 error.login.title = Błąd logowania
-error.login.description = W trakcie łączenia z „%1$S” Sync napotkał błąd. Spróbuj ponownie.
+error.login.description = W trakcie łączenia z „%1$S” napotkano błąd. Spróbuj ponownie.
 error.login.prefs.label = Ustawienia…
 error.login.prefs.accesskey = U
 # should decide if we're going to show this
 error.logout.title = Błąd podczas wylogowania
@@ -22,6 +22,6 @@
 error.login.prefs.label = Ustawienia…
 error.login.prefs.accesskey = U
 # should decide if we're going to show this
 error.logout.title = Błąd podczas wylogowania
-error.logout.description = W trakcie łączenia Sync napotkał błąd. Prawdopodobnie wszystko jest w porządku i nie musisz podejmować żadnego działania.
+error.logout.description = W trakcie łączenia napotkano błąd. Prawdopodobnie wszystko jest w porządku i nie musisz podejmować żadnego działania.
 error.sync.title = Błąd podczas synchronizacji
@@ -27,6 +27,6 @@
 error.sync.title = Błąd podczas synchronizacji
-error.sync.description = W trakcie synchronizacji Sync napotkał błąd: %1$S i automatycznie podejmie ponowną próbę synchronizacji.
-error.sync.no_node_found = Serwer Sync jest w tej chwili zajęty, ale nie musisz nic robić. Synchronizacja danych rozpocznie się, gdy tylko będzie to możliwe!
+error.sync.description = W trakcie synchronizacji napotkano błąd: %1$S. Zostanie automatycznie podjęta ponowna próba synchronizacji.
+error.sync.no_node_found = Serwer synchronizacji jest w tej chwili zajęty, ale nie musisz nic robić. Synchronizacja danych rozpocznie się, gdy tylko będzie to możliwe!
 error.sync.no_node_found.title = Opóźnienie synchronizacji
 error.sync.serverStatusButton.label = Stan serwera
 error.sync.serverStatusButton.accesskey = n
@@ -30,8 +30,8 @@
 error.sync.no_node_found.title = Opóźnienie synchronizacji
 error.sync.serverStatusButton.label = Stan serwera
 error.sync.serverStatusButton.accesskey = n
-error.sync.needUpdate.description = Aby kontynuować synchronizowanie danych, musisz zaktualizować Firefox Sync.
-error.sync.needUpdate.label = Aktualizuj Firefox Sync
+error.sync.needUpdate.description = Aby kontynuować synchronizację danych, musisz zaktualizować usługę synchronizacji.
+error.sync.needUpdate.label = Aktualizuj usługę synchronizacji
 error.sync.needUpdate.accesskey = A
 error.sync.tryAgainButton.label = Synchronizuj teraz
 error.sync.tryAgainButton.accesskey = S

